### ISSUE TITLE
New content and options

### ISSUE THREAD

**Issue by mvuorre** [CrumpLab/vertical#51]
It might be useful to list more options for content (e.g. slides, manuscript templates, ...) and somehow instruct how different options could be used. Even if we didn't include them in *vertical*, it would be nice to list them somewhere. 

e.g. For experiments, there's [psychtestr](https://pmcharrison.github.io/psychTestR/index.html).

**Comment by CrumpLab** [comment]
Good idea, we should probably also refer to psychtestr in a revision of the manuscript, looks like a great package.

**Comment by CrumpLab** [comment]
I took a stab at adding a new vignette that lists currently suggested R Packages and alternatives. It's now live on the master branch.
